### ISSUE TITLE
Allow editing a card from the Card Browser "Preview" window

### ISSUE THREAD

**Issue by david-allison-1** [ankidroid/Anki-Android#7026]
Just came up in the [Google Group](https://groups.google.com/g/anki-android/c/UggNrUCGzzI) that editing a card from "Browse" was not intuitive, this seems like it will make "Edit" both more useful and more accessible

**Comment by mikehardy** [comment]
Yeah - same person on Play Store review comments and I asked them to open an issue because I liked the idea. I'll cross-link them

**Comment by david-allison-1** [comment]
More complex than the following as elements can now be deleted while being/pending in the preview queue. Needs tests to ensure good behaviour if the current card, adjacent cards, or all cards are deleted.

```
    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
        if (item.getItemId() == R.id.action_edit) {
            editCard();
            return true;
        } else {
            return super.onOptionsItemSelected(item);
        }
    }


    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        getMenuInflater().inflate(R.menu.previewer, menu);
        return super.onCreateOptionsMenu(menu);
    }
```

**Comment by david-allison-1** [comment]
Remaining follow-on with Unit tests, I'll get it done tomorrow:

> EDIT: Actually... this might be possible if the template is changed, which might cause a re-render... or if you have 2 templates, 1 card and delete the template with the card. Let me check. Nice catch!

**Comment by david-allison-1** [comment]
Fixed in #7042
